src/background.js

```diff
@@ -159,6 +159,9 @@ browser.runtime.onMessage.addListener(function(eventData) {
     case 'metrics-limit-reached':
       sendMetrics('limit-reached', eventData.context);
       break;
+    case 'metrics-context-menu':
+      sendMetrics('context-menu', eventData.context);
```

Comment:
Need to add this new event to https://github.com/mozilla/notes/blob/master/docs/metrics.md